### DISCUSSION TITLE
Data-aware scheduling question

### DISCUSSION THREAD

**Discussion by matthieucx** [apache/airflow#30881] [Q&A] [answered]
Hello! I've had some trouble finding the info online, does anyone know what happens in this situation.

Let's say we have:
Dag A produces -> Dataset Z -> Consumed by Dag B 
What happens if a run of Dag A finishes while Dag B is running ? Does another run of Dag B is triggered/scheduled ?

Thanks in advance!

**Comment by potiuk** [comment, marked as answer]
yes.

**Reply by matthieucx** [reply]
Thanks!
